### ISSUE TITLE
[META] Moving questions from the issues to GitHub Discussions

### ISSUE THREAD

**Issue by sm-Fifteen** [tiangolo/fastapi#2447]
(not following the issue template since this is not actually an issue regarding FastAPI)

So FastAPI has been maintaining a pretty active community over the past 2 years and I know from the past propositions to move community questions to Stack Overflow (#992, #2228) that the general concensus is that there are benefits in keeping these close to the git repo. It does have the downside of making the issues a fair bit more clogged than is convenient, and I know GitHub introduced a [forum-like "Discussions" tab](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions) for repo owners to enable that has been [other repos](https://github.com/ImageMagick/ImageMagick/discussions) have the kind of built-in forum we've been using the issues like for a while.

Would there be any interest in enabling this for the FastAPI repo so we can move new questions and feature discussions there?

**Comment by sm-Fifteen** [comment]
@tiangolo: I hadn't heard about this until recently, and this sounds like the type of thing that would be of interest to you, given what you were saying in #992.

**Comment by Kludex** [comment]
About three weeks ago, I had a conversation with @tiangolo and we discussed about chats/forums (it was when the discord was created). At the time he was favorable towards the GH discussions, but he pointed out that the feature was in **close beta** stage, so there was nothing we can do about it. 

P.S.: if it's freely available now, we can 😅

@sm-Fifteen is it freely available now?

**Comment by sm-Fifteen** [comment]
Huh, I was under the impression that it was just a settings toggle away, but looking at it more closely, it does appear that you need to apply for the limited/"private" beta [here](https://github.community/t/can-one-apply-for-github-discussions-beta/121120), by the looks of it.

**Comment by ArcLightSlavik** [comment]
![Screenshot 2020-12-01 073434](https://user-images.githubusercontent.com/31127044/100701127-ad996b80-33a7-11eb-8e5c-aeda8b33c5b1.png)

I think if it would be a more public beta you would have the option show up on the list (screenshot above). 
So it's probably still a closed/private beta?

**Comment by yareyaredesuyo** [comment]
Looks like discussion page is opened.

How to use discussion features from here? 

https://github.com/tiangolo/fastapi/discussions

**Comment by Kludex** [comment]
@yareyaredesuyo Could you rephrase your question? :thinking:

**Comment by yareyaredesuyo** [comment]
@Kludex 

Moving questions and another things from issues to GitHub Discussions?

Like: 

- questions (issues) ⇨ Q & A (discussion) 
- enhancement (issues) ⇨ Ideas (discussion) 

And prohibit to make QA related things from issues ?

**Comment by ArcLightSlavik** [comment]
![Screenshot 2020-12-07 103002](https://user-images.githubusercontent.com/31127044/101327326-349e8600-3877-11eb-9cbb-e804d1648025.png)
^ copying here so there's no confusion

**Comment by tiangolo** [comment]
Thanks for the conversation here everyone! :coffee: 

Yeah, my plan is to move questions to GitHub Discussions once there's support for templates and an API.

Meanwhile, it's easier to keep questions mainly in issues. I'm still leaving GitHub Discussions enabled, but more as an experiment for now than the "official" questions place. Just to see how many people end up there and how it evolves.

**Comment by sm-Fifteen** [comment]
Glad that's been answered!
